OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -107,8 +107,8 @@ public string TerrainType(CPos cell)
 			return Context.World.Map.GetTerrainInfo(cell).Type;
 		}
 
-		[Desc("Returns true if there is only one human player.")]
-		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.IsSinglePlayer; } }
+		[Desc("Returns true if there is only one human client.")]
+		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.NonBotClients.Count() == 1; } }
```

Comment:
I think we actually want to have `NonBotPlayers` here. The presence of potential specs won't be of any interest to scripts.